Backend/Remora.Discord.Gateway/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -48,12 +49,14 @@ public static class ServiceCollectionExtensions
     /// <param name="serviceCollection">The service collection.</param>
     /// <param name="tokenFactory">A function that retrieves the bot token.</param>
     /// <param name="buildClient">Extra options to configure the rest client.</param>
+    /// <param name="maxDispatchItems">How many items can be concurrently queued for dispatch.</param>
     /// <returns>The service collection, with the services added.</returns>
     public static IServiceCollection AddDiscordGateway
     (
         this IServiceCollection serviceCollection,
         Func<IServiceProvider, string> tokenFactory,
-        Action<IHttpClientBuilder>? buildClient = null
+        Action<IHttpClientBuilder>? buildClient = null,
+        int maxDispatchItems = 100
```

Comment:
Don't take this as a parameter here - let the consumer use `Configure` if they want to change it.

Reply:
Okay, I see what you mean.